README.md

```diff
@@ -63,61 +65,62 @@ A file server that uses the network file system (NFS) protocol is the minimum re
+#### Jump Virtual Machine
+
+The jump VM or bastion server can manage NFS folders if you provide SSH access to it. The jump VM must have the NFS storage mounted to it at `<JUMP_SVR_RWX_FILESTORE_PATH>`. If you want to manage the NFS server yourself, the jump VM is not required. Here is the required folder structure for the jump VM:
 
   ```bash
   <JUMP_SVR_RWX_FILESTORE_PATH>     <- mounted NFS export
     /pvs                            <- location for persistent volumes
     /<namespace>                    <- folder per namespace
       /bin                          <- location for open source directories
-      /data                         <- location for SAS and CAS Data
+      /data                         <- location for SAS and CAS data
       /homes                        <- location for user home directories to be mounted
-      /astores                      <- location for astores
+      /astores                      <- location for ASTORES
   ```
 
 ## Getting Started
 
-### Clone this project
+### Clone this Project
 
-Run these commands in a Terminal session:
+Run the following commands in a terminal session:
 
 ```bash
-# clone this repo
+# clone this repository
 git clone https://github.com/sassoftware/viya4-deployment
 
 # move to directory
 cd viya4-deployment
 ```
 
-### Authenticating Ansible to access Cloud Provider
+### Authenticating Ansible to Access Cloud Provider
 
 See [Ansible Cloud Authentication](./docs/user/AnsibleCloudAuthentication.md) for details.
 
-**NOTE:** At this time, only required for GCP with external postgres
+**NOTE:** At this time, additional setup is only required for GCP with external PostgreSQL.
 
 ### Customize Input Values
 
-The playbook uses ansible vars for configuration. It is recommended to encrypt both this file and the other configs (sitedefault, kubeconfig, and tfstate) using [ansible vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html).
+The playbook uses Ansible variables for configuration. SAS recommends that you encrypt both this file and the other configuration files (sitedefault, kubeconfig, and tfstate) using [Ansible Vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html).
 
-#### Ansible Vars
+#### Ansible Vars File
 
-The ansible vars file is the main configuration file. Create a file named ansible-vars.yaml to customize any input variable value. Example variable definition files are provided in the ./examples folder. For more details on the supported variables refer to [CONFIG-VARS.md](docs/CONFIG-VARS.md).
+The Ansible vars.yaml file is the main configuration file. Create a file named ansible-vars.yaml to specify values for any input variables. Example variable definition files are provided in the `./examples` folder. For more details on the supported variables, refer to [CONFIG-VARS.md](docs/CONFIG-VARS.md).
 
-#### Sitedefault
+#### (Optional) Sitedefault File
```

Comment:
Do you normally have Optional before the heading? I usually see this after like

#### Sitedefault File (Optional)

Reply:
We place it before the heading in the docs. For ex., see https://go.documentation.sas.com/doc/en/calcdc/3.5/dplyml0phy0lax/n0w37tlhoww4wmn1s1w563uo23mn.htm